### ISSUE TITLE
Negative numbers in line graph

### ISSUE THREAD

**Issue by kpcyrd** [yaronn/blessed-contrib#134]
Hey :)

I'm trying to build a line graph that can have negative numbers. After some troubleshooting I figured out that everything below 0 is cut off. I'm currently adding a 2nd number to everything to ensure the final number is positive as a workaround. This is somewhat quirky since I have to do some math while reading the graph.

Can a flag be introduced so the graph grows into negative numbers the same way it grows when increasing positive numbers?

**Comment by zlx362211854** [comment]
I have the same problem.

**Comment by aureooms** [comment]
Same for bar charts

**Comment by theBliz** [comment]
got the same issue. Did anyone find a solution? @kpcyrd

**Comment by kpcyrd** [comment]
@theBliz No. The numbers I had didn't go below -3000 so I've just added 3000 to every number and added an extra line in a different color that's always 3000 to indicate zero.

**Comment by theBliz** [comment]
Not really doable for my use case. Ended up writing a small chart page in Html

**Comment by comster** [comment]
I just submit a PR that solved this for me.

**Comment by devamin** [comment]
Hi any update on this

**Comment by comster** [comment]
@devamin It would be cool if my PR was merged in, but you can checkout my fork until then.

**Comment by yrammos** [comment]
In a similar vein, I need bar graphs with negative values (or otherwise expanding downwards). 

Awestruck by this library. Thank you, @yaronn.
